tests/integration/components/frost-modal-input-test.js

```diff
@@ -9,18 +10,48 @@ describeComponent(
     integration: true
   },
   function () {
+    let container, application
+
+     beforeEach(function () {
+       Ember.run(() => {
+         application = Ember.Application.create()
+         container = application.__container__
+         application.deferReadiness()
+       initialize(container, application)
+      })
+    })
+
     it('renders', function () {
-      // Set any properties with this.set('myProperty', 'value')
```

Comment:
cannot seem to get this to pass. error - `Assertion Failed: The key provided to set must be a string, you passed undefined`